pynest/examples/sinusoidal_poisson_generator.py

```diff
@@ -56,7 +56,8 @@
 
 nest.SetKernelStatus({'resolution': 0.01})
 
-g = nest.Create('sinusoidal_poisson_generator', n=2,
+num_nodes = 2
+g = nest.Create('sinusoidal_poisson_generator', n=num_nodes,
                 params=[{'rate': 10000.0,
```

Comment:
Maybe use
```
params={'rate': [10000.0, 0.0], 'amplitude': [5000.0, 10000.0], 'frequency': [10.0, 5.0], 'phase': [0.0, 90.0]}
```
instead?

Reply:
Here it's a complete inversion, but I think the arrays for each parameter might be the more frequent use-case. So changing here too.
@sarakonradi, we could include this conversion explicititly in the API documentation of `nest.Create()`... what do you think? (in another PR)

Reply:
@sarakonradi, is there another PR already?

Reply:
Just created an Issue on this! See #1813.